enterprise/internal/campaigns/store_changesets.go

```diff
@@ -405,6 +408,57 @@ type ListChangesetsOpts struct {
 	OnlyWithoutDiffStats bool
 	OnlySynced           bool
 	ExternalServiceID    string
+	TextSearch           []ListChangesetsTextSearchExpr
+}
+
+type ListChangesetsTextSearchExpr struct {
+	Term string
+	Not  bool
+}
+
+func (expr ListChangesetsTextSearchExpr) query() *sqlf.Query {
+	// The general query format is for a positive query is:
+	//
+	// (field ~* value OR field ~* value)
```

Comment:
Do you mean this?

```suggestion
	// (field1 ~* value OR field2 ~* value)
```

Asking because I'm not sure I understand the queries in the docstring here :|